Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob.Tests/UnitTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using NUnit.Framework;
+using System;
+using System.IO;
+
+namespace Frends.AzureBlobStorage.ReadBlob
+{
+    [TestFixture]
+    public class ReadTest
+    {
+        private static Source _source, _sourcefail;
+
+
+        [OneTimeSetUp]
+        public static void Setup()
+        {
+            var containername = "test";
+            var blobName = "test.txt";
+            var sastoken = "sp=r&st=2022-04-06T10:17:16Z&se=2022-04-11T18:17:16Z&spr=https&sv=2020-08-04&sr=b&sig=aG5N%2BywcDxb01QF9VZAUiEwTniUawJHZd4OsMFf21ow%3D";
+            var uri = $"https://testsorage01.blob.core.windows.net/{containername}/{blobName}?";
+            var localConnectionString = "UseDevelopmentStorage=true";
```

Comment:
Credentials should be provided in environment variable, so they can be managed in the CI. Here's an example: https://github.com/FrendsPlatform/Frends.AzureBlobStorage/blob/main/Frends.AzureBlobStorage.UploadBlob/Frends.AzureBlobStorage.UploadBlob.Tests/UnitTests.cs#L17